completions/_quick-lint-js

```diff
@@ -0,0 +1,22 @@
+#compdef _quick-lint-js quick-lint-js
```

Comment:
It can be named `quick-lint-js.zsh` but it's suggested to use `_quick-lint-js` according to  [zsh-users HowTo](https://github.com/zsh-users/zsh-completions/blob/master/zsh-completions-howto.org#telling-zsh-which-function-to-use-for-completing-a-command). I'm ok with changing it if you are!

> Completion functions for commands are stored in files with names beginning with an underscore _

Reply:
Filename kept as `_quick-lint-js` and the file header has now following comments:
> This script is used to provide command line completion for
> quick-lint-js when using zsh shell